bundle-3.2/pom.xml

```diff
@@ -0,0 +1,1279 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2017 ABSA Group Limited
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>spark-3.2-spline-agent-bundle_2.12</artifactId>
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>za.co.absa.spline.agent.spark</groupId>
+        <artifactId>spline-spark-agent_2.12</artifactId>
+        <relativePath>../pom.xml</relativePath>
+        <version>0.5.6-kensu-spark20fix</version>
+    </parent>
+
+    <dependencies>
```

Comment:
FYI all stuff bellow in this file is only copy paste from https://github.com/AbsaOSS/spline-spark-agent/blob/release/0.7/bundle-3.2/pom.xml